public_html/wp-content/themes/wordcamp-central-2012/style.css

```diff
@@ -1203,6 +1204,17 @@ li:after { content: "."; display: block; height: 0; clear: both; visibility: hid
 	padding-bottom: 20px;
 }
 
+/* !Featured Sponsor */
+.wordcamp-mes-sponsors > li {
+	width: 100%;
+	/*min-height: 300px;
```

Comment:
And here too— if we need to bring back the code, we can grab it from revision history.

Reply:
Fair point, I can remove it all.

Reply:
91c2d44b